content/spire/try/getting-started-k8s.md

```diff
@@ -29,15 +29,15 @@ The steps in this guide have been tested on these Kubernetes versions: 1.13.1, 1
 If you are using Minikube to run this tutorial you should specify some special flags as described [here](#minikube).
 {{< /info >}}
 
-# Obtain the Required Files {#section-1}
+## Obtain the Required Files
```

Comment:
Not quite sure why these are now L2 headings? Doesn't this move everything under the single Overview heading?

Reply:
Fixed. While trying to removed those `section-x` references I did not notice I ended up changing the headings. Thanks for pointing that Andrew.